gini/gini-switch-sdk-ios
# Added swiftlint and fixed all warnings and errors

# Introduction

Added swift lint to the project. Also removed all warnings and errors. But I also added some exceptions to the configuration file that should have been there. Namely the `colon` one.

# How to test

Build the project and see that there are no warnings.
Try to add two new lines to a file and verify that you get a lint warning. Or something similar. Make sure you have `swiftlint` installed.

# Merge info

Automatic
